testapps/testapp/src/main/res/raw/msal_config_default.json

```diff
@@ -2,6 +2,7 @@
   "client_id" : "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0",
   "authorization_user_agent" : "DEFAULT",
   "redirect_uri" : "msauth://com.msft.identity.client.sample.local/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
+  "handle_null_taskaffinity": true,
```

Comment:
I set this to true in the test app since i modified the manifest.  We can chat about how to test this scenario correctly.

Reply:
We should add a test with manual steps in our test suite.

Reply:
Will do.